datus/cli/_cli_utils.py

```
@@ -0,0 +1,88 @@
+from rich.console import Console
+
+from datus.utils.loggings import get_logger
+
+logger = get_logger(__name__)
+
+
+def prompt_input(
+    console: Console,
+    message: str,
+    default: str = "",
+    choices: list = None,
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)